src/io/flutter/performance/FrameRenderingDisplay.java

```diff
@@ -1,9 +1,9 @@
 /*
- * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Copyright 2020 The Chromium Authors. All rights reserved.
```

Comment:
Not critical, but in the future we can leave copyright years as is for existing files.

Reply:
I think this happened automatically when I moved the file. Will watch for this in the future.